### DISCUSSION TITLE
User specific default root folder

### DISCUSSION THREAD

**Discussion by delmlund** [sct/overseerr#1731] [Feature Requests]
Hello,

An idea that would really streamline my requests would be user specific default root folder option. 

Ex.

user 1: defaults to “main” root folder. 
user 2: defaults to “hidden” root folder. 

I have my plex setup with “hidden” folders. So the if wife, in-laws (user 1 in this example) request something it goes to normal general folder. 

But if people from my work request something it goes to a hidden folder (user 2) on plex that the first group can’t see. 

I currently cannot have the user 2 group auto-approve since it will go to main folder. So each request has to be edited before accepting. 

Not sure if there currently a solution that would do this or this is a valid suggestion. 

Awesome app, keep up the good work.

**Comment by TheCatLady** [comment]
Hi @delmlund!  We already have an open feature request for this. You can follow #609 for updates. :)
